spec/support/authenticated_session_helper.rb

```diff
@@ -1,12 +1,14 @@
 # frozen_string_literal: true
 
 module AuthenticatedSessionHelper
-  def sign_in(user = FactoryBot.build(:user, :loa3), token = nil, raw = false)
+  def sign_in(user = FactoryBot.build(:user, :loa3), token = nil, raw = false, pact = false)
     user = user.persisted? ? user : User.create(user)
     token ||= 'abracadabra'
     session_object = Session.create(uuid: user.uuid, token: token)
     session_options = { key: 'api_session', secure: false, http_only: true }
-    if raw
+    if pact
+      session_object
+    elsif raw
```

Comment:
Not for today, because it would introduce side-effects: But someday™, I think it'd be neat of this method `yield if block_given?` instead of passing raw & pact flags.